common/lib/capa/capa/responsetypes.py

```diff
@@ -1083,16 +1255,80 @@ def late_transforms(self, problem):
         self.do_shuffle(self.xml, problem)
         self.do_answer_pool(self.xml, problem)
 
-    def get_score(self, student_answers):
+    def grade_via_points(self, **kwargs):
```

Comment:
They should be off by one line from each other (this line 1258 is old line 1257) until the OptionResponse function (which starts on line 1575). After OptionResponse comes NumericalResponse, which picks up at line 1788 in the old one and line 1649 in this one.

I grabbed the raw of this one and my old one, and had TextWrangler compare them for me. The only differences were in OptionResponse and some `get_inner_html_from_xpath` changes that are from someone else.